deploy/helm/flightctl/values.yaml

```
@@ -23,18 +25,21 @@
 ## @param global.internalNamespace Namespace where internal components are deployed
 ## @param global.clusterLevelSecretAccess Allow flightctl-worker to access secrets at the cluster level for embedding in device configs
 ## @param global.appCode This is only related to deployment in Red Hat's PAAS.
-## @param global.exposeServicesMethod How the FCTL services should be exposed. Can be either nodePort or route
+## @param global.exposeServicesMethod How the FCTL services should be exposed. Can be either nodePort, route or gateway
 ## @param global.nodePorts Node port numbers for FCTL services
-
+## @param hubConfig.ocpIngress Same as global.baseDomain but this property is expected to be set by ACM operator
+## @param hubConfig.apiUrl Same as global.auth.openShiftApiUrl but this property is expected to be set by ACM operator
 
 global:
   target: "standalone" # standalone, acm
   baseDomain: ""
+  ocpIngress: "" # alternative to baseDomain used by the multiclusterhub operator
   baseDomainTls:
     cert: ""
     key: ""
   storageClassName: ""
   imagePullSecretName: ""
+  apiUrl: "" # alternative used by multiclusterhub operator
```

Is the alternative to `externalOpenShiftApiUrl`? 
May be good to show it here too (as you already did for `ocpIngress`)

---

I would actually try to put these MCH operator stuff next to each other
```suggestion
  ## @section multiclusterhub operator
  ocpIngress: "" # alternative to baseDomain
  apiUrl: "" # alternative to auth.k8s.externalOpenShiftApiUrl
  
```

---

Done.

---

Is `ocpIngress` not used anymore by the MCE operator?

---

OK, I saw the PR in the ACM side. Seems fine now.